Rendering fixes for thumbnail on work details page

nicktiberi opened the pull request:
- short titles + descriptions now show up to the right of the thumbnail
- the thumbnail is hidden on small screen sizes

### Long Title
<img width="1463" alt="Screen Shot 2022-03-09 at 10 49 03 AM" src="https://user-images.githubusercontent.com/639920/157477621-d64ef7b9-f9a7-4454-84d4-2b01d6b0f3cf.png">

### Short Title
<img width="1463" alt="Screen Shot 2022-03-09 at 10 49 11 AM" src="https://user-images.githubusercontent.com/639920/157477691-a154821b-aa8b-43ee-82a9-faa5b1fb24d0.png">

### No thumbnail
<img width="1470" alt="Screen Shot 2022-03-09 at 10 50 10 AM" src="https://user-images.githubusercontent.com/639920/157477796-e8a9ef54-d1f4-4403-89d5-ae6ec67676ca.png">

### Wrapping
<img width="789" alt="Screen Shot 2022-03-09 at 10 52 24 AM" src="https://user-images.githubusercontent.com/639920/157478274-1137a36e-bb2b-461c-8002-61aaea94b061.png">

### Mobile View
<img width="543" alt="Screen Shot 2022-03-09 at 10 52 30 AM" src="https://user-images.githubusercontent.com/639920/157478304-45b56a8d-7f9e-4d3b-9e7f-41726c0a78d3.png">

awead commented:
@cam156 No, it passes locally. However, I'm also seeing the same failure in one of the other PRs, so it's possible this is some kind of transient error that was introduced with a prior PR.

nicktiberi committed:
Rendering fixes for thumbnail on work details page

- short titles + descriptions now show up to the right of the thumbnail
- the thumbnail is hidden on small screen sizes

ajkiessl reviewed (approved):
Sweet!  Looks good.  Gosh, I gotta get better at this stuff 🧠 🔨 .
